src/config/logger.ts

```diff
@@ -110,6 +111,7 @@ interface LoggerOptions {
 export const getLogger = (name: string, options: LoggerOptions = {}): Logger => {
 	return createLogger(merge<Logger.LoggerOptions, LoggerOptions>({
 		name,
+		id: newUUID(),
```

Comment:
I think this is not needed because we add one in frontend-middleware-logger, don't we?